### PR TITLE
Connection Page UI setup

### PR DESCRIPTION
**Pull request by S3784866**
Set up the Connection pages UI.

### CODE REVIEW

File: Dashboard/Views/ConnectionView.axaml.cs [hunk 1]
```diff
@@ -9,7 +14,48 @@ public partial class ConnectionView : UserControl
...
+            Title = "Select Log File",
+            AllowMultiple = false
+        });
 
+        if (files.Count >= 1)
+        {
+            var filePath = files[0].Path.LocalPath;
+            LogFileSelected.Text = filePath;
+        }
+    }
```
Comment (AriadneLovelace) [inline Dashboard/Views/ConnectionView.axaml.cs:36]: Is this the input used to replay a scrutineering run?

File: Dashboard/Views/ConnectionView.axaml [hunk 2]
```diff
@@ -1,8 +1,64 @@
...
+              Background="#c4c4c4">
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:ConnectionViewModel />
+    </Design.DataContext>
+    <Panel>
+        <ThemeVariantScope RequestedThemeVariant="Light">
+        <!-- Main Content Grid -->
+        <Grid RowDefinitions="Auto,Auto">
```
Comment (AriadneLovelace) [inline Dashboard/Views/ConnectionView.axaml]: Looks good :) maybe just center the inputs so its the middle of the screen, similar to status and scrutineering page

File: Dashboard/Views/ConnectionView.axaml.cs (hunk 1 above)
Comment (AriadneLovelace) [reply Dashboard/Views/ConnectionView.axaml.cs:36]: after launching tool it is haha

File: Dashboard/Views/ConnectionView.axaml (hunk 2 above)
Comment (S3784866) [reply Dashboard/Views/ConnectionView.axaml]: Can do.
https://github.com/nickowen44/AV-Datalogger/pull/15/commits/0524be42bccec030cc15bde8f9ad52fcac5a4e11

**Pull request merged**